pkg/cmd/pulumi/destroy.go

```diff
@@ -130,7 +131,11 @@ func newDestroyCmd() *cobra.Command {
 			}
 
 			proj, root, err := readProject()
-			if err != nil {
+			if err != nil && errors.Is(err, workspace.ErrProjectNotFound) {
+				// Allow destroy outside of a project using backend to resolve.
+				proj = &workspace.Project{}
+				root = ""
```

Comment:
logging.Warningf the text about we couldn't find the project but we're carrying on anyway

Reply:
👍 good call